### PR TITLE
Houdini: Add geometry check for VDB family

### PR DESCRIPTION
**Pull request by mustafa-zarkash**
## Changelog Description
When `sop_path` on Geometry ROP node points to a non SopNode, this validator `validate_vdb_output_node.py` will error and crash when this line is executed
`sop_node.geometryAtFrame(frame)`

## Additional info
The bug addressed here is the same as in this [PR](https://github.com/ynput/OpenPype/pull/5230)

### CODE REVIEW

File: openpype/hosts/houdini/plugins/publish/validate_vdb_output_node.py [hunk 1]
```diff
@@ -55,6 +55,8 @@ def get_geometry_at_frame(sop_node, frame, force=True):
     """Return geometry at frame but force a cooked value."""
     with update_mode_context(hou.updateMode.AutoUpdate):
         sop_node.cook(force=force, frame_range=(frame, frame))
+        if not hasattr(sop_node, "geometry"):
+            return
```
Comment (BigRoy) [inline openpype/hosts/houdini/plugins/publish/validate_vdb_output_node.py:59]: Can we do this check outside of the with statement? That way we don't force a cook and change update states for no reason.

Comment (mustafa-zarkash) [reply openpype/hosts/houdini/plugins/publish/validate_vdb_output_node.py]: good observation! 
thank you

**Review by BigRoy — APPROVED** [review]
Looks good to me.

**Review by moonyuet — APPROVED** [review]
LGFM

**Pull request merged**